### PR TITLE
Drive train/#45 auto align vision

### PR DESCRIPTION
**Pull request by deslobodzian**
Does not have straight PID

### CODE REVIEW

File: src/main/java/org/frc5687/infiniterecharge/robot/OI.java [hunk 1]
```diff
@@ -91,8 +91,8 @@ public void initializeButtons(Shifter shifter, DriveTrain driveTrain, Turret tur
         _operatorRightBumper.toggleWhenPressed(new ShootSpeedSetpoint(shooter, this, 1.0));
         _operatorRightTrigger.whenHeld(new Shoot(shooter, indexer, turret, this));
 
-        _driverLeftBumper.whenPressed(new RaiseIntake(intake));
-        _driverRightBumper.whenPressed(new LowerIntake(intake));
+//        _driverLeftBumper.whenPressed(new RaiseIntake(intake));
+//        _driverRightBumper.whenPressed(new LowerIntake(intake));
```
Comment (BenBernardCIS) [inline src/main/java/org/frc5687/infiniterecharge/robot/OI.java:95]: Did you mean to leave these commented out?

File: src/main/java/org/frc5687/infiniterecharge/robot/Constants.java [hunk 2]
```diff
@@ -24,8 +24,8 @@
         public static final double ROTATION_SENSITIVITY = 0.75;
 
         public static final double CREEP_FACTOR = 0.25;
-        public static final double LEFT_DISTANCE_PER_PULSE = 0.0286206896551724;
-        public static final double RIGHT_DISTANCE_PER_PULSE = 0.0286206896551724;
+        public static final int CPR = 8192;
```
Comment (BenBernardCIS) [inline src/main/java/org/frc5687/infiniterecharge/robot/Constants.java:27]: What does CPR stand for?

Comment (deslobodzian) [reply src/main/java/org/frc5687/infiniterecharge/robot/Constants.java:27]: Counts Per Revolution, number is taken from Rev Robotics

File: src/main/java/org/frc5687/infiniterecharge/robot/OI.java (hunk 1 above)
Comment (deslobodzian) [reply src/main/java/org/frc5687/infiniterecharge/robot/OI.java:95]: These will be changed to shift. I'm pretty sure they are changed in #50

**Review by BenBernardCIS — APPROVED** [review]

**Pull request merged**